tsconfig.json

```diff
@@ -0,0 +1,26 @@
+{
+  "compilerOptions": {
```

Comment:
They're the CRA default ones, I think they're pretty good, is there any option you had in mind that's missing here?

Reply:
Nope, if those are the defaults, that LGTM! Otherwise, I was gonna ask questions around things like why we need to allow js, etc, but I assume it's probably best not to venture away from the CRA defaults unless we have a good reason!